LayoutTests/platform/mac-wk1/TestExpectations

```diff
@@ -2530,7 +2530,7 @@ imported/w3c/web-platform-tests/geolocation-API/permission.https.html [ WontFix
 http/tests/websocket/tests/hybi/multiple-connections-limit.html [ Skip ]
 
 # IPC is for WK2
-ipc/shared-video-frame-size.html [ Skip ]
+ipc [ Skip ]
```

Comment:
It is slightly better to mark it as WontFix, because in theory, skipped tests are tracked as something that needs to be fixed. In practice, there is currently no difference, as we don't actively try to unskip.